src/hooks/useGlobalAppState.ts

```diff
@@ -32,6 +36,10 @@ const handleWakeUp = async (
   }
 
   await handleSecurityFlowWakeUp(client)
+
+  // On appStart the homeview will hide the splashscreen
+  // On wakeup, it is more ambiguous so we force it
+  if (!isAppStart) await hideSplashScreen(lockScreens.LOCK_SCREEN)
```

Comment:
I'm wondering if this `hideSplashScreen(lockScreens.LOCK_SCREEN)` should be called in `LockScreenWrapper` instead.
We can put it next to `hideSplashScreen()` so we hide both the initial SplashScreen and the LockScreen one.

I think this should work as other "non lock" scenario already hide this splashscreen on `SecurityService:61` and `SecurityService:207`